src/Middleware/UserPageDecorationMiddleware.php

```diff
@@ -89,6 +89,9 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
                 $this->page
             );
         }
+        if (method_exists($page, 'getBreadcrumbs')) {
```

Comment:
I don't think reflection is required here. NDB_Page always has getBreadcrumbs defined, you just need to make sure it's not null.

Reply:
But is $page always an NDB_Page?

Reply:
Oh! you mean to check that $page is not null?